frontend/src/workspace/buildingtools/FunNode.js

```diff
@@ -111,52 +120,36 @@ function FunNode(props) {
     );
   }
 
-// +----------------------------+
-// | Trashcan                   |
-// +----------------------------+------------------------------------
-
-  useEffect(() => {
-    if (!props.renderFunction) {
-      setShowImage(false);
-    }
-  }, [props.renderFunction]); //Re-renders whenever the render function changes
+  // +----------------------------+
+  // | Trashcan                   |
+  // +----------------------------+------------------------------------
```

Comment:
There is a set of redundant Trashcan comments here. If this was intended can you make these more different?